src/pages/Pomodoro/lib/SettingsContext.tsx

```diff
@@ -1,12 +1,11 @@
 import {createContext, useState} from "react";
-// import style from "../Logic/Settings.module.css";
 
-export const SettingContext = createContext({});
+export const SettingContext = createContext();
 
 const SettingsContextProvider = (props) => {
 
     const [pomodoro, setPomodoro] = useState(0);
-    // this if from the settings page
+    // this is from the settings page
     const [executing, setExecuting] = useState({});
```

Comment:
this var name is also a bit off, cause it's an Object, but it's called like a verb